First_section.html

```diff
@@ -278,7 +278,37 @@ <h1 class="language">Languages</h1>
         </ul>
       </div>
 
-      <section class="footer"></section>
+      <div class="footer">
+    <img class="Img" src="contact form shapes-mobile.svg" alt="footer Image">
+      <h4> Contact me</h4>
+      <p id="text">If you have an application you are interested in developing, 
+        a feature that you need built or a project that needs coding. 
```

Comment:
- [ ] I appreciate your work, you have respected the project requirements, and your code looks fine, but we have a few points of improvement to raise up before the work goes approved, please take a look at this image below to compare your work with the original design.
- [ ] On the top-left side, as pointed out with the red arrow, please modify that section to look like the original design.
- [ ] Your paragraph text is not centered as the one in the original design.
- [ ] Please add some padding on the left and top of your placeholder as shown in the design.
- [ ] We have some white space on the left and the right of this section, as I highlighted with the red arrow, please make a few touches to match the correct alignment as per the design.

<img width="737" alt="5" src="https://user-images.githubusercontent.com/77843050/207356160-bab6d9c0-ba91-4520-a4c1-da78126ea0bd.png">

Reply:
thank you for informative feedback @joskalenda  . I have corrected everything according your recommendation. The white space on the left and right side are due to default margin applied on the body. it is varies depend on the browser or live view application you are using. fortunately, i removed the margin in css. hope it is will reflect on your side too.